copier/xattrs.go

```diff
@@ -16,7 +16,9 @@ const (
 )
 
 var (
-	relevantAttributes = []string{"security.capability", "security.ima", "user.*"} // the attributes that we preserve - we discard others
+	relevantAttributes    = []string{"security.capability", "security.ima", "user.*"} // the attributes that we preserve - we discard others
+	initialXattrListSize  = 64 * 1024
```

Comment:
Should change regarding extended attributes list size belong in a different commit. Although change is trivial so its a non-blocker.